modules/execute-in-vm/pkg/execute/executor.go

```diff
@@ -248,7 +248,7 @@ func (e *Executor) ensureVMStop() error {
 		e.attemptedStop = true
 
 		log.Logger().Debug("stopping a vm", zap.String("name", vmName), zap.String("namespace", vmNamespace))
-		if err := e.kubevirtClient.VirtualMachine(vmNamespace).Stop(vmName, &kubevirtv1.StopOptions{}); err != nil {
+		if err := e.kubevirtClient.VirtualMachine(vmNamespace).Stop(context.TODO(), vmName, &kubevirtv1.StopOptions{}); err != nil {
```

Comment:
Are you planning to change the context or should we just use `context.Background()`?

Reply:
I can do a follow up PR where I change the context type